index.js

```diff
@@ -33,7 +33,7 @@ instance.prototype.init = function() {
 
 	self.states = {}
 	self.init_feedbacks()
-

```

Comment:
This is my mistake but I thik these should have a ; after them.

Reply:
I was curious about that, fixed.